lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -41,11 +41,15 @@ qx.Class.define("callbackery.ui.form.Auto", {
      * @param layout {Incstance} qooxdoo layout instance
      * @param formRenderer {Class} qooxdoo form render class
      */
-    construct : function(structure, layout, formRenderer) {
+    construct : function(structure, layout, formRenderer, plugin) {
```

Comment:
please add documentation for the new parameter

Reply:
Done